AIMoba/AIMoba/Controllers/GameController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AIMoba.Controllers
+{
+    public class GameController : Controller
+    {
+        public IActionResult Game()
+        {
+            return View();  //Megjeleníti az oldalt.
+        }
+        
+        public class Player
+        {
+            public int ID { get; set; } //a játékos azonosítója
+            public int score { get; set; }  //a játékos lépéseinek száma
+
+            public Player(int ID = 0, int score = 0)
+            {
+                this.ID = ID;
+                this.score = score;
+            }
+
+            public void NextMove()
+            {
+                //játékos lépése
+            }
+
+        }
+        public class Robot : Player
+        {
+            public int difficulty { get; private set; } //a robot nehézségi szintje később állítható lesz
+
+            public Robot(int id = 0, int score = 0, int diff = 1)
+            {
+                this.ID = id;
+                this.score = score;
+                this.difficulty = diff;
+            }
+
+            public void NextRobotMove()
+            {
+                //robot lépése
+            }
+        }
+
+        public static void ShowWinner(int currentPlayer, int score)
+        {
+            //eldönti, hogy ki nyert, és megjeleníti.
+        }
+
+        public static void RefreshPage()
+        {
+            //TODO: Oldal frissítése
+        }
+
+        static void Main(string[] args)
+        {
+            GridModel grid;
+
+            Player[] players = new Player[playerNumber];
+            players[playerNumber] = new Robot();
+
+            for (int i = 0; i < playerNumber - 1; i++)
```

Comment:
Próbáld meg linq-val csinálni. Pl.: players.foreach(i->i.ID ...

Reply:
Megírtam linq-val.

Reply:
Látom, de bent martadt a ciklus is. Akkor most kétszer csinálja meg ugyan azt?

Reply:
A ciklus csak figyelmetlenségből maradt bent, javítottam.